### PR TITLE
movement controlled by backend

### PR DESCRIPTION
**Pull request by neu5**

### CODE REVIEW

File: backend/test/test.js [hunk 1]
```diff
@@ -1,54 +1,56 @@
 const chai = require("chai");
 const chaiHttp = require("chai-http");
-const io = require("socket.io-client");
+// const io = require("socket.io-client");
```
Comment (przenz) [inline backend/test/test.js:3]: Please cleanup all commented out code.

File: src/scenes/GameScene.js [hunk 2]
```diff
@@ -32,6 +32,8 @@ export default class Game extends Phaser.Scene {
   create() {
     this.buildMap();
 
+    // this.extractMapFromPhaserObject();
```
Comment (przenz) [inline src/scenes/GameScene.js:35]: Remove.

File: src/scenes/GameScene.js [hunk 3]
```diff
@@ -47,13 +49,24 @@ export default class Game extends Phaser.Scene {
...
     this.groundLayer = this.tilemap.createLayer("Ground", tilesetOutside);
-    this.tilemap.createLayer("Buildings", tilesetOutside);
+    this.tilemap.createLayer("OnGround", tilesetOutside);
+    this.tilemap.createLayer("Collides", tilesetOutside);
     this.tilemap.createLayer("Above", tilesetOutside);
+  }
 
-    this.socket = io();
+  extractMapFromPhaserObject() {
+    // find better way to do that
```
Comment (przenz) [inline src/scenes/GameScene.js:58]: Please create an issue and add it here like this:
```suggestion
    // TODO (#xx) find better way to do that
```

**Review by przenz — APPROVED** [review]

File: .prettierignore [hunk 4]
```diff
@@ -1 +1,2 @@
 .travis.yml
+backend/map.js
```
[the remark this answers is not in the document]
Comment (neu5) [reply .prettierignore:2]: Yup, but first we need to have generic way to extract map. I did it manually this time.

File: backend/test/test.js (hunk 1 above)
Comment (neu5) [reply backend/test/test.js:3]: I wish to fix the tests rather than removing them :D

**Pull request merged**